### PR TITLE
(student/06-writing-proposal): clarify and link formal rules

### PR DESCRIPTION
**Pull request by kevinnls**
the portion about "visa" is a bit ambiguous for
GSoC 2022, where general public contributors are
welcomed.

patch to clarify that and also link to official rules

### CODE REVIEW

**Review by rspier — CHANGES_REQUESTED** [review]
Thanks!

File: pages/student/06-writing_a_proposal.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This is a competitive program, each year Google turns down many more applicants
 
 ## The Basics
 
-First and foremost, make sure you meet Google's formal requirements for participation in Summer of Code. Also be sure you are eligible to participate in a program like GSoC with your visa, etc. Hopefully, you have already checked this by now, but be sure to double-check before you waste time and energy on a proposal.
+First and foremost, make sure you meet [Google's formal requirements][rules] for participation in Summer of Code. Also, if you are studying overseas, ensure that you are eligible to participate in a program like GSoC with your visa, etc. Hopefully, you have already checked this by now, but be sure to double-check before you waste time and energy on a proposal.
```
[the remark this answers is not in the document]
Comment (kevinnls) [reply pages/student/06-writing_a_proposal.md:13]: i had been wondering the same

hmmm, taking inspiration from the official [rules][1]
> for the duration of the Program, be eligible to work in the country in which they reside; and

what do you think of changing the entire line to
> Additionally, ensure that you are eligible to work in the country where you reside for the duration of the program.

[1]: https://summerofcode.withgoogle.com/rules#:~:text=for%20the%20duration%20of%20the%20Program%2C%20be%20eligible%20to%20work%20in%20the%20country%20in%20which%20they%20reside%3B%20and

[the remark this answers is not in the document]
Comment (rspier) [reply pages/student/06-writing_a_proposal.md]: I think that's clearer.

**Comment by kevinnls** [comment]
look good?

**Review by rspier — APPROVED** [review]

**Comment by rspier** [comment]
Looks good.

**Pull request merged**